src/main/java/jpa/domain/Line.java

```diff
@@ -0,0 +1,47 @@
+package jpa.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import jpa.infrastructure.jpa.BaseEntity;
+
+/**
+ * @author : leesangbae
+ * @project : jpa
+ * @since : 2020-12-15
+ */
+
+@Entity
+public class Line extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(unique = true)
+    private String name;
+
+    private String color;
+
+    public Line() {
```

Comment:
```suggestion
    protected Line() {
```
기본생성자는 사용하지 않으면 최대한 감추는게 좋습니다
하지만 jpa에서 기본생성자를 사용하니 다른 객체가 최대한 접근 못하게 protected로 막아주면 좋겠네요

Reply:
다른 클래스도 같이 정리해주세요